src/test/util/setup_common.h

```diff
@@ -87,8 +87,8 @@ static constexpr CAmount CENT{1000000};
 /** Initialize Dash-specific components during chainstate initialization (NodeContext-friendly aliases) */
 void DashChainstateSetup(ChainstateManager& chainman,
                          node::NodeContext& node,
-                         bool fReset,
-                         bool fReindexChainState,
+                         bool llmq_dbs_in_memory,
+                         bool llmq_dbs_wipe,
```

Comment:
nit; it's a bit risky to change one bool flag to other with completely different meaning; but so far as it is DashChainstateSetup I think it's not a big deal.